test/fixtures/rules/std_rule_cases/L999_set_language_statement.yml

```diff
@@ -0,0 +1,51 @@
+rule: L999
```

Comment:
Hey, for changes to the dialect, please add fixtures to the yml test cases in https://github.com/sqlfluff/sqlfluff/tree/main/test/fixtures/dialects/tsql